app/css/grid.scss

```diff
@@ -0,0 +1,27 @@
+@import '~@openmrs/style-referenceapplication/src/sass/fragments/grid';
+
+.grid-row {
+    @include row();
+}
+
+.button-row {
+    @include row();
+    margin-top: 1%;
+}
+
+@for $i from 1 through 12 {
+    .column-#{$i} {
+        @include column($i);
+        margin-right: 1em;
+    }
+}
+
+@for $i from 1 through 12 {
+    .pull-right-#{$i} {
+        @include push($i);
+    }
+}
+
+input[type=number]{
```

Comment:
I would suggest you use a class to define this property rather than making it global for all input fields of type number

Reply:
Thank you, working on it now.